test/integration/client-side-encryption/driver.test.ts

```diff
@@ -471,3 +694,589 @@ describe('Range Explicit Encryption with JS native types', function () {
     });
   });
 });
+
+describe('CSOT', function () {
+  describe('Auto encryption', function () {
```

Comment:
Should this be nested above? unusual to have multiple top-level suites.

Reply:
The previous block says "CSOT on client encryption", so should this be "CSOT on auto encryption"? What is the difference between what the previous block tests vs this one? ("client encryption" is a bit general)

Reply:
Because we parallelized this work the tests merged as separate blocks. I thought I could wholesale merge this suite with the one above but there is mixed nesting that would need to be undone. 

This CSOT block covers Aditi's AutoEncryption, state machine, and createEncryptedCollection changes, the one above is all the other helpers on ClientEncryption that I worked on. So I would want to move createEncryptedCollection into that block and de-nest auto and statemachine.

But there's no actual test changes so let's follow up on this.